src/openzaak/utils/inclusion.py

```diff
@@ -0,0 +1,222 @@
+                ).data
+            result.setdefault(model_key, []).append(data)
+        # in-place sort of inclusions
+        for value in result.values():
+            value.sort(key=sort_key)
+        return result
+
+    def _instance_inclusions(
+        self, path, serializer, instance, inclusion_serializers=None
+    ):
+        # Use inclusion serializers derived from parent serializer
+        inclusion_serializers = inclusion_serializers or getattr(
+            serializer, "inclusion_serializers", {}
+        )
+        for name, field in serializer.fields.items():
+            for entry in self._field_inclusions(
+                path, field, instance, name, inclusion_serializers
+            ):
+                yield entry
+
+    def _field_inclusions(self, path, field, instance, name, inclusion_serializers):
+        # if this turns out to be None, we don't want to do a thing
+        if instance is None:
+            return
+        new_path = path + (name,)
+        if isinstance(field, BaseSerializer):
+            for entry in self._sub_serializer_inclusions(new_path, field, instance):
+                yield entry
+            return
+        inclusion_serializer = inclusion_serializers.get(name)
+        if inclusion_serializer is None:
+            return
+        if isinstance(inclusion_serializer, str):
+            inclusion_serializer = import_string(inclusion_serializer)
+        for obj in self._some_related_field_inclusions(
+            new_path, field, instance, inclusion_serializer
+        ):
+            yield obj, inclusion_serializer
+            # when we do inclusions in inclusions, we base path off our
+            # parent object path, not the sub-field
+
+            # TODO option to derive serializers from parent serializer, instead of child
+            # serializer
+            nested_serializers = {
+                field_name[len(name) + 1 :]: serializer
+                for field_name, serializer in inclusion_serializers.items()
+                if field_name.startswith(name)
+            }
+
+            nested_path = (
+                new_path if self.nested_inclusions_use_complete_path else new_path[:-1]
+            )
+            for entry in self._instance_inclusions(
+                nested_path,
+                inclusion_serializer(instance=object),
+                obj,
+                inclusion_serializers=nested_serializers,
+            ):
+                yield entry
+
+    def _some_related_field_inclusions(
+        self, path, field, instance, inclusion_serializer
+    ):
+        if self.allowed_paths is not None and path not in self.allowed_paths:
+            return []
+
+        # Properly include loose fk fields
+        if (
+            isinstance(field, FKOrURLField)
+            or isinstance(field, LooseFKHyperlinkedRelatedField)
+            or isinstance(field, LooseFKHyperlinkedIdentityField)
```

Comment:
```suggestion
            or isinstance(field, LooseFKHyperlinkedRelatedField)
```
`LooseFKHyperlinkedIdentityField` is a subclass of `LooseFKHyperlinkedRelatedField` already, so that should match.